addons/crm/tests/test_crm_lead_convert.py

```diff
@@ -8,6 +8,10 @@
 
 @tagged('lead_manage')
 class TestLeadConvert(crm_common.TestLeadConvertCommon):
+    """
+    TODO: created partner (handle assignation) has team of lead
+    TODO: create partner has user_id  coming from wizard
```

Comment:
allez, do it so it's behind us after :D

Reply:
Plein le luc comme on dit. Ca sera pour plus tarte.